single_tech_samples/datafactory/sample2_data_pre_processing_with_azure_batch/README.md

```diff
@@ -0,0 +1,160 @@
+# Running container workloads from Azure Data Factory on Azure Batch <!-- omit in toc -->
+
+## Contents <!-- omit in toc -->
+
+- [1. Solution Overview](#1-solution-overview)
+  - [1.1. Scope](#11-scope)
+  - [1.2. Use Case](#12-use-case)
+  - [1.3. Architecture](#13-architecture)
+  - [1.4. Technologies used](#14-technologies-used)
+- [2. How to use this sample](#2-how-to-use-this-sample)
+  - [2.1. Prerequisites](#21-prerequisites)
+    - [2.1.1 Software Prerequisites](#211-software-prerequisites)
+  - [2.2. Setup and deployment](#22-setup-and-deployment)  
+  - [2.3. Deployment validation and Execution](#23-deployment-validation-and-execution)
+  - [2.4. Clean-up](#24-clean-up)
+
+## 1. Solution Overview
+
+This solution demonstrates how we can perform data pre-processing by running Azure Batch container workloads from Azure Data Factory.
+
+This sample will focus on provisioning an Azure Batch account, Azure Data Factory and other required resources where you can run the ADF pipeline to see how the data pre-processing can be done on huge volume of data in an efficient and scalable manner.  
```

Comment:
I think you should also highlight _why_ this particular sample architecture of setting up Azure batch with ADF is super useful to get around the limitations of ADF and Azure Batch orchestration, such as the slow spin-up times (thus need for containers), and the lack of ADF support for Azure Batch containers.